core/services/run_manager_test.go

```diff
@@ -607,27 +608,27 @@ func TestRunManager_Create_fromRunLogPayments(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			config, configCleanup := cltest.NewConfig(t)
 			defer configCleanup()
+			config.Set("DATABASE_TIMEOUT", "10s") // Lots of parallelized tests
```

Comment:
Still feels hacky to me :/

Reply:
Totally agree, but the whole issue seems to be that the Postgres advisory lock implementation is timeout-based, and the tests are running long enough to cause it to time out.  The advisory lock exists so that two nodes (one being a failover) pointed at the same DB can't simultaneously access it, and without building complex infrastructure to coordinate between them more elegantly, I'm afraid we might be stuck with this.  Open to suggestions, though.  If you want more info on this problem, @jleeh can probably lend some insight.

Reply:
Is the advisory lock actually still failing?

Reply:
In a way, yes, but not due to a defect in it.  I think what's happening is that this test is taking too long to run, and so the advisory lock timeout is tripped.  Do you think it's something else?